README.md

```diff
@@ -199,3 +199,8 @@ meta = gp.GPTableMeta(table_output_name, table_output_schema, columns_output_typ
 GPDatabase_instance.check_table_if_exist(table_name, schema)
 
 ```
+## PR
+
+- install dev requirements `pip install -r requirements-dev.txt`
+- `black` the code use `black .  --line-length 100`
```

Comment:
Doesn't black have a config file so we can put it into the root of project?
I really need my editor be aware of the `100` limit

Reply:
Yes it has config using `toml` https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html

For now, only one rule I think its not necessary to add new file~

Reply:
please add it, I really need the lint config to make my editor happy. Thanks!

Reply:
added